src/main/java/org/broadinstitute/hellbender/tools/walkers/featuremapping/AddFlowSNVQualityArgumentCollection.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.hellbender.tools.walkers.featuremapping;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.Hidden;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Set of arguments for the {@link FlowFeatureMapper}
+ */
+public class AddFlowSNVQualityArgumentCollection implements Serializable{
+    private static final long serialVersionUID = 1L;
+
+    public enum SnvqModeEnum {
+        Legacy,
+        Optimistic,
+        Pessimistic,
+        Geometric
+    };
+
+    /**
+     *  maximum value for
+     *  delta in score
+     **/
+    @Argument(fullName = "limit-phred-score", doc = "Limit value for phred scores", optional = true)
+    public double limitPhredScore = Double.NaN;
+
+    /**
+     *  include duplicate read?
+     **/
+    @Argument(fullName = "include-dup-reads", doc = "include duplicate reads", optional = true)
+    public boolean includeDupReads = false;
+
+    /**
+     *  keep supplementary alignments?
+     **/
+    @Argument(fullName = "keep-supplementary-alignments", doc = "keep supplementary alignments ?", optional = true)
+    public boolean keepSupplementaryAlignments = false;
+
+    /**
+     * snvq computation mode
+     */
+    @Argument(fullName = "snvq-mode", doc = "ksnvq computation mode", optional = true)
```

Comment:
Maybe some more details on the strategies?

Reply:
these have been experimental strategies at computing the snvq. We have now settled on the "geometric" strategy - all other code has been erased. There is no need need to explain since it is simply the way snvq is computed.